### PR TITLE
[READY] Add test for action

### PR DESCRIPTION
**Pull request by tofufighting**
This adds a workflow that calls this action, so that we can run the action in PRs to test it.

### CODE REVIEW

File: .github/workflows/self_test.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Test This Action
```
Comment (tofufighting) [inline .github/workflows/self_test.yml:1]: self-test

Comment (tofufighting) [reply .github/workflows/self_test.yml:1]: self-test

Comment (tofufighting) [reply .github/workflows/self_test.yml:1]: self-test

Comment (tofufighting) [inline .github/workflows/self_test.yml:1]: self-test should be successful

**Comment by tofufighting** [comment]
Test run worked as expected: https://github.com/Nebulaworks/action-standardize-pr-comment/actions/runs/3184459818/jobs/5192885376

**Comment by tofufighting** [comment]
self-test this won't work

**Review by sarcasticadmin — APPROVED** [review]
@tofufighting looks great, thanks for including this so we have a easy way to test this going forward :shipit:

**Pull request merged**